### ISSUE TITLE
Identical total exp but optimizer picked slower route

### ISSUE THREAD

**Issue by madness800** [Infiziert90/SubmarineTracker#10]
Hi,

Not sure if it's a bug or intended but the optimized route picked the slower route despite the total exp yield is identical on both route.

This is the mogship link to cross reference the information
https://www.mogship.com/submarines/exp-calculator?rank=97&slot0=23&slot1=24&slot2=25&slot3=10&excludeSectors=50,84,85,86,87,88,89,90,91,92,93&excludeMaps=5
 
Suggested route & duration
![image](https://github.com/Infiziert90/SubmarineTracker/assets/144722345/252e93ce-c0ac-487e-a75d-d1e09d062bac)

Faster time by a few mins
![image](https://github.com/Infiziert90/SubmarineTracker/assets/144722345/e5cabc2a-4a22-4abb-b9d7-c4fa32c063e7)

Many thanks and keep up with the good work!

**Comment by Infiziert90** [comment]
> Hi,
> 
> Not sure if it's a bug or intended but the optimized route picked the slower route despite the total exp yield is identical on both route.

The plugin picks routes based on the distance first, so if both have 185, it will just one of the two and go with it
Maybe i will fix it in the future but the time save is so minimal that it prolly isn't even worth it

**Comment by madness800** [comment]
Yeah, like you said it is minimal if it was anything outrages then it probably needs to be addressing. I figured to report it so you're aware, since I'm not sure if it was bug, intentional or an oversight.

However, the dev inside of me screams out I should make an issue on it. xD

**Comment by Infiziert90** [comment]
![image](https://github.com/Infiziert90/SubmarineTracker/assets/22648449/542d6605-171a-4770-952c-1dbaa5d1217a)
Sorts it correctly now, 42:08:11
